tensorflow_lite_support/ios/task/processor/utils/sources/TFLDetectionUtils.m

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#import "tensorflow_lite_support/ios/task/processor/utils/sources/TFLDetectionUtils.h"
+
+@implementation TFLDetectionUtils
+
++ (TFLDetectionResult *)detectionResultFromCDetectionResults:
+    (TfLiteDetectionResult *)cDetectionResult {
+  if (cDetectionResult == nil) return nil;
+
+  NSMutableArray *detections = [[NSMutableArray alloc] init];
+  for (int i = 0; i < cDetectionResult->size; i++) {
+    TfLiteDetection cDetection = cDetectionResult->detections[i];
+    NSMutableArray *categories = [[NSMutableArray alloc] init];
+    for (int j = 0; j < cDetection.size; j++) {
+      TfLiteCategory cCategory = cDetection.categories[j];
```

Comment:
TfLiteCategory -> TFLCategory conversion is also used in image classification. You should refactor it into a common method in a class category similarly to `TFLBaseOptions (Helpers)` and use in both use cases. With that, it'll be easier in the future if we add parameters to TfLiteCategory.

Reply:
Fixed